spec/imsc-hrm.html

```diff
@@ -216,12 +216,18 @@ <h2>Documentation Conventions</h2>
 
   <section id='terms'>
     <h2>Terms and Definitions</h2>
-      <p><dfn>IMSC Document Instance</dfn>. A <dfn data-lt="Document Instance|Document Instances"
-      data-cite="ttml2#terms-document-instance">Document Instance</dfn> that conforms to any profile defined in any edition of
-      [[!IMSC]].</p>
-      
-      <p><dfn>empty ISD</dfn>. an <a>Intermediate Synchronic Document</a> with no <dfn
-      data-cite="IMSC#dfn-presented-region">presented region</dfn>.</p>
+    
+    <p><dfn>character</dfn>. The character code property of a <dfn data-cite="ttml2#terms-character-information-item">Character Information Item</dfn>.</p>
+
+    <p class="note">The term <a>character</a> is for practical purposes
+      the same as a [=code point=].</p>
+
+    <p><dfn data-lt="code point|code points">code point</dfn>. As defined by [[i18n-glossary]].</p>
```

Comment:
If _code point_ is defined, then all instances of _code point_ should be replaced with `<a>code point</a>`

Reply:
In fact, no point in defining in "terminology", since the first instance of _code point_ can use: `<dfn data-lt="i18n-glossary/#ef_code_point>code point</dfn>`."

See `<dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Documents</dfn>` as an example.

Reply:
See [previous discussion](https://github.com/w3c/imsc-hrm/pull/42#discussion_r811941924) : defining in this way prevents easy traversal from the definition to the places where the term is used, because clicking on it takes you straight to the defining document.

Reply:
Addressed in https://github.com/w3c/imsc-hrm/commit/ee04753e86c97b5930744c4351e5f440ebd0176e though I note that the two additional places where code point is used, in the definitions of GCpy and Ren in §9 Paint Text, arguably should be "character" instead. Thoughts?

Reply:
> See [previous discussion](https://github.com/w3c/imsc-hrm/pull/42#discussion_r811941924) : defining in this way prevents easy traversal from the definition to the places where the term is used, because clicking on it takes you straight to the defining document.

Ok. I see pluses and minuses. I think we should be consistent however. I have so far avoided adding terms in the definition section that identical to terms defined in the other documents.

Reply:
Changing back to "character" as discussed on the TTWG call today.

Reply:
Addressed in a16bbf1